### PR TITLE
F/W table and stored procedures.

### PR DESCRIPTION
**Pull request by cmcheung-UCS**
Signed-off-by: Ming Cheung <chi.m.cheung@intel.com>

### CODE REVIEW

File: data/db/DAI-Volt-Procedures.sql [hunk 1]
```diff
@@ -706,6 +706,19 @@ CREATE PROCEDURE FROM
...
+
+CREATE PROCEDURE FROM
+    CLASS com.intel.dai.procedures.FwVersionDump;
+
+CREATE PROCEDURE FROM
+    CLASS com.intel.dai.procedures.FwVersionHistoryInsert;
+
+CREATE PROCEDURE FROM
+    CLASS com.intel.dai.procedures.FwVersionHistoryDump;
+
```
Comment (paul-amonson) [inline data/db/DAI-Volt-Procedures.sql:721]: If these are not complicated my preference would be not to use Java procedures but SQL procedures.

Comment (cmcheung-UCS) [reply data/db/DAI-Volt-Procedures.sql:721]: Don't you want things to be consistent?  Also, I think voltdb tend to have poor non-Java SQL support.

File: procedures/src/main/java/com/intel/dai/procedures/FwVersionHistoryDump.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package com.intel.dai.procedures;
+
+import org.voltdb.SQLStmt;
+import org.voltdb.VoltProcedure;
+import org.voltdb.VoltTable;
+
+public class FwVersionHistoryDump extends VoltProcedure {
+    private static final String SQL_TEXT =
+            "SELECT * FROM FW_Version WHERE ID STARTS WITH ? ORDER BY DbUpdatedTimestamp;";
```
Comment (nbwhite) [inline procedures/src/main/java/com/intel/dai/procedures/FwVersionHistoryDump.java:9]: This is identical to FwVersionDump. Is that intentional?

File: data/db/DAI-Volt-Procedures.sql (hunk 1 above)
Comment (cmcheung-UCS) [reply data/db/DAI-Volt-Procedures.sql:721]: Good catch.  I will change it;

**Pull request merged**